content/homework.rst

```diff
@@ -123,8 +150,13 @@ You shouldn't be dependent on work for social interactions.
 Communication
 =============
 
-Use "away" and "busy" functionality of you instant messaging tool to let other
-know about your status.
+Use "away" and "busy" functionality of you instant messaging tool to let other know about your status.
+
+Use a dedicated account for work and log in with that account
+only when you are at work.
+On the same line, join the team IRC chat only when you are *at work*.
```

Comment:
I don't agree 100% with this one. I know we can leave and re-join on return and check the logs but I prefer to check the channel window, it's easier.

We can mark ourselved away just as you said above. Not sure if this works when someone is writting to you on the channel (ex alibotean: check this out)